qubes/firewall.py

```diff
@@ -422,9 +503,9 @@ def load_v1(self, xml_root):
         policy_v1 = xml_root.get('policy')
         assert policy_v1 in ('allow', 'deny')
         if policy_v1 == 'allow':
-            self.policy = Action('accept')
+            policy = Action('accept')
         else:
-            self.policy = Action('drop')
+            policy = Action('drop')
```

Comment:
Do I understand this correctly: this is how firewall model used to work and this code is used to load policies written by previous Qubes releases?

Is `policy` variable used as this object? Wouldn't a simple boolean flag suffice?